packages/create-cloudflare/e2e-tests/frameworks.test.ts

```diff
@@ -357,15 +357,20 @@ function getFrameworkTests(opts: {
 			svelte: {
 				promptHandlers: [
 					{
-						matcher: /Which Svelte app template/,
+						matcher: /Which template would you like/,
```

Comment:
Why this change?

Reply:
These changes are to do with the sv package, not create-cloudflare.